API_Quick_Start.md

```diff
@@ -0,0 +1,267 @@
+# Quickstart
+
+This quickstart guide shows a real use case scenario of interaction with the API in an Apiculture context.
+
+## Prepare your environment
+
+The provided examples make use of the HTTPie command line tool (installation instructions: https://httpie.org/docs#installation)
+
+## Provide credentials
+
+- Calls to The API must include an `Authorization` header containing a Bearer access token. It takes the following form:
+
+```
+Authorization: Bearer <access token>
+```
+
+An access token can be obtained in two ways:
+
+- If a client has its service account enabled, an access token can be obtained with the following request:
+
+```
+http --form POST https://data-hub.eglobalmark.com/auth/realms/datahub/protocol/openid-connect/token client_id=<client_id> client_secret=<client_secret> grant_type=client_credentials
+```
+
+- If a client wants to make API calls on behalf of an end user, an access token can be obtained in exchange of the authorization code contained in the redirect URL after a user authenticates on the authentication server. This process, called the Authorization Code Flow, is described exhaustively in the OpenID Connect specification: https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth.
+
+One simple way to have the access token without copy-pasting is to keep it in a variable:
+
+```
+export TOKEN=$(http --form POST https://data-hub.eglobalmark.com/auth/realms/datahub/protocol/openid-connect/token client_id=<client_id> client_secret=<client_secret> grant_type=client_credentials | jq -r .access_token)
+```
+
+Then to simply use it the HTTP requests:

```

Comment:
I think a word is missing

Reply:
I don't think so.
is to keep it in a variable, then to simply use it the HTTP requests.

Reply:
It at least misses an `in` (to simply use it **in** the HTTP requests)